examples/atlas-datalake-pipeline/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "mongodbatlas_project" "atlas-project" {
+  org_id = var.atlas_org_id
+  name   = var.atlas_project_name
+}
+
+resource "mongodbatlas_cluster" "automated_backup_test" {
+  project_id                  = mongodbatlas_project.atlas-project.id
+  name                        = var.cluster_name
+  provider_name               = "GCP"
+  provider_region_name        = "US_EAST_4"
+  provider_instance_size_name = "M30"
```

Comment:
Most of our examples are M10 - any reason for an M30 here?

Reply:
no, I will update them. Thanks